src/main/java/edu/ucsb/cs156/spring/backenddemo/controllers/TidesController.java

```diff
@@ -2,8 +2,43 @@
 
 import org.springframework.web.bind.annotation.RestController;
 
+import edu.ucsb.cs156.spring.backenddemo.services.TidesQueryService;
+import lombok.extern.slf4j.Slf4j;
 
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+
+@Api(description="Tides from https://api.tidesandcurrents.noaa.gov/api/prod/")
+@Slf4j
 @RestController
+@RequestMapping("/api/tides")
 public class TidesController {
+    
+    ObjectMapper mapper = new ObjectMapper();
+
+    @Autowired
+    TidesQueryService tidesQueryService;
+
+    @ApiOperation(value = "Get water level for date range, in local time.")
```

Comment:
I think that the `@ApiOperation` annotation should have another argument:
`value = "For station id, see: https://tidesandcurrents.noaa.gov/tide_predictions.html?gid=1393"`

Reply:
@eduglt added in the notes section, which I think is consistent with what they have in the UI. lmk if you have anything else